mako.1.scd

```diff
@@ -183,96 +183,15 @@ dismissed with a click or via *makoctl*(1).
 
 	Default: none
 
-# CRITERIA-ONLY STYLE OPTIONS
+# CONFIGURATION FILE
 
-Some style options are not useful in the global context and therefore have no
-associated command-line option.
+The mako configuration file is located at *~/.config/mako/config* or at
+*$XDG\_CONFIG\_HOME/mako/config*.
 
-*invisible* 0|1
-	Whether this notification should be invisible even if it is above the
-	_max-visible_ cutoff. This is used primarily for hiding members of groups.
-	If you want to make more than the first group member visible, turn this
-	option off within a _group-index_ criteria.
+The configuration file supports *CRITERIA* with allows changing behavior
+per-notifiaction, per-application, or on other conditions.
```

Comment:
Typos: "with allows", "notifiaction".

This makes it sound like you can only use criteria in the config file, and not also the globals. How about something like "in addition to the same configuration supplied by the command line flags, the configuration file also allows overriding styles based on the source and content of each notification".